### PR TITLE
Kisilov_Vadim

### PR DESCRIPTION
**Pull request by Kisilov-Vadim**

### CODE REVIEW

File: src/numJewelsInStones.js [hunk 1]
```diff
@@ -28,7 +28,13 @@
...
  */
 function numJewelsInStones(jewels, stones) {
-  // write code here
+  let coinsidenceCount = 0;
+  for (let i = 0; i < stones.length; i++) {
+    if (jewels.includes(stones[i]) && stones[i].match(/[a-z]/gi)) {
+      coinsidenceCount++;
+    }
+  }
+  return coinsidenceCount;
```
Comment (gestap) [inline src/numJewelsInStones.js]: new line before return
```suggestion

  return coinsidenceCount;
```

File: src/numJewelsInStones.js [hunk 2]
```diff
@@ -28,7 +28,13 @@
  * @return {number}
  */
 function numJewelsInStones(jewels, stones) {
-  // write code here
+  let coinsidenceCount = 0;
+  for (let i = 0; i < stones.length; i++) {
```
Comment (gestap) [inline src/numJewelsInStones.js]:
```suggestion

  for (let i = 0; i < stones.length; i++) {
```

File: src/numJewelsInStones.js (hunk 1 above)
Comment (Kisilov-Vadim) [reply src/numJewelsInStones.js]: okey! Thanks.

File: src/numJewelsInStones.js [hunk 3]
```diff
@@ -28,7 +28,15 @@
  * @return {number}
  */
 function numJewelsInStones(jewels, stones) {
-  // write code here
+  let coinsidenceCount = 0;
```
Comment (gestap) [inline src/numJewelsInStones.js]:
```suggestion
  let coincidenceCount = 0;
```

File: src/numJewelsInStones.js [hunk 4]
```diff
@@ -28,7 +28,15 @@
...
+
+  for (let i = 0; i < stones.length; i++) {
+    if (jewels.includes(stones[i]) && stones[i].match(/[a-z]/gi)) {
+      coinsidenceCount++;
```
Comment (gestap) [inline src/numJewelsInStones.js]:
```suggestion
      coincidenceCount++;
```

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**